orchestration/dagster_orchestration/hca_orchestration/tests/test_solids.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from unittest.mock import Mock, patch
+
+from dagster import execute_solid
+
+from hca_orchestration.solids import post_import_validate
+
+
+class SolidsTestCase(unittest.TestCase):
```

Comment:
❓ Are there any error cases we can exercise here (i.e., what happens if the client 500s, etc.)

Reply:
perhaps? I think we might need to add a bunch of tests for hca-utils so that we test it's functionality there, and don't test the functionality here. I guess we'd still want to handle this suggested error case though. I'll add something.

Reply:
I added a ticket to write tests for hca-utils https://broadinstitute.atlassian.net/browse/DSPDC-1570